core/src/org/otw/open/dto/DrawablePixmap.scala

```diff
@@ -46,6 +46,8 @@ class DrawablePixmap(val pixmapMask: Pixmap) extends Disposable {
     **/
   def isTransparent: Boolean = {
     var maskAlphaSum = 0
+    val pixmapSize = pixmapMask.getWidth * pixmapMask.getHeight
+    val percentageRequired = pixmapSize * 0.5 / 100
```

Comment:
can you please extract this in a constant? or pass it as a parameter to isTransparent